docs/README.md

```diff
@@ -1,12 +1,12 @@
-# Microsoft Azure Container Service Engine - Builds Docker Enabled Clusters
+# AKS-Engine - Builds Docker Enabled Clusters
 
 ## Overview
 
 This cluster definition examples demonstrate how to create a customized Docker Enabled Cluster on Microsoft Azure.
 
 ## User Guides
 
-* [AKS Engine](acsengine.md) - shows you how to build and use the AKS engine to generate custom Docker enabled container clusters
+* [AKS Engine](aksengine.md) - shows you how to build and use the AKS engine to generate custom Docker enabled container clusters
```

Comment:
Slight preference for `AKS-Engine` instead of `the AKS engine`

Reply:
I preferred "AKS Engine" to refer to the project and "aks-engine" for the command-line tool. But I see @jackfrancis prefers "AKS-Engine," and I'm ok with that. I just want to see consistency.